core/server/common/src/main/java/alluxio/RpcUtils.java

```diff
@@ -31,17 +30,17 @@
    * @param callable the callable to call
    * @param <T> the return type of the callable
    * @return the return value from calling the callable
-   * @throws AlluxioTException if the callable throws an Alluxio or runtime exception
+   * @throws AlluxioTException if the callable throws an exception
    */
   public static <T> T call(Logger logger, RpcCallable<T> callable) throws AlluxioTException {
     try {
       return callable.call();
     } catch (AlluxioException e) {
       logger.debug("{}, Error={}", callable, e.getMessage());
-      throw e.toThrift();
-    } catch (Exception e) {
+      throw AlluxioStatusException.fromAlluxioException(e);
+    } catch (RuntimeException e) {
```

Comment:
`catch` blocks only catch exceptions thrown in the body of their corresponding `try`